### ISSUE TITLE
Crash when pinning a time type dimension

### ISSUE THREAD

**Issue by l2dy** [allegro/turnilo#283]
```
Error: Expected StringFilterClause, got: RelativeTimeFilterClause { [redacted...], duration: P1D } dimension-tile.tsx:498:12
	DimensionTile.prototype.prepareRows dimension-tile.tsx:498:12
	DimensionTile.prototype.render dimension-tile.tsx:591:17
	finishClassComponent react-dom.development.js:7873
	updateClassComponent react-dom.development.js:7850
	beginWork react-dom.development.js:8225
	performUnitOfWork react-dom.development.js:10224
	workLoop react-dom.development.js:10288
	callCallback react-dom.development.js:542
	invokeGuardedCallbackDev react-dom.development.js:581
	invokeGuardedCallback react-dom.development.js:438
	renderRoot react-dom.development.js:10366
	performWorkOnRoot react-dom.development.js:11014
	performWork react-dom.development.js:10967
	requestWork react-dom.development.js:10878
	scheduleWorkImpl react-dom.development.js:10732
	scheduleWork react-dom.development.js:10689
	scheduleTopLevelUpdate react-dom.development.js:11193
	updateContainer react-dom.development.js:11231
	renderSubtreeIntoContainer react-dom.development.js:15229
	render react-dom.development.js:15290
	<anonymous> main.tsx:77:2
	<anonymous> self-hosted:1018:17
	run es6.promise.js:75
	notify/< es6.promise.js:92
	flush _microtask.js:18
```

**Comment by adrianmroz** [comment]
I think it's resolved by #272 

We don't support non StringFilters in Pinboard. But maybe we should handle such error better? React Error Boundaries - https://reactjs.org/docs/error-boundaries.html?

**Comment by l2dy** [comment]
> I think it's resolved by #272

It works! I only checked for duplicate issues, but forgot to check PRs.

> We don't support non StringFilters in Pinboard. But maybe we should handle such error better? React Error Boundaries - https://reactjs.org/docs/error-boundaries.html?

Exactly. I'm aware that it's unsupported, but the crash is not user-friendly.

**Comment by adrianmroz** [comment]
Let's proceed in generic issue #306.
